lib/internal/Magento/Framework/Model/EntitySnapshot/AttributeProvider.php

```diff
@@ -53,31 +54,29 @@ public function __construct(
      * Returns array of fields
      *
      * @param string $entityType
-     * @return array
+     * @return string[]
      * @throws \Exception
      */
-    public function getAttributes($entityType)
+    public function getAttributes($entityType): array
```

Comment:
Unfortunately, we are not allowed to change public method signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)